README.md

```diff
@@ -44,12 +44,14 @@ metadata:
   name: pomerium
   annotations:
     ingressclass.kubernetes.io/is-default-class: "false"
+    ingress.pomerium.io/default-cert-secret: pomerium/wildcard-localhost-pomerium-io
```

Comment:
let's not place this annotation in the example, as people might think it's default behaviour (and it's not).

Reply:
OK. So where/how _should_ it be documented?